clusterloader2/pkg/config/templatefunctions.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"math/rand"
+	"text/template"
+)
+
+// GetFuncs returns map of names to functions, that are supported by template provider.
+func GetFuncs() template.FuncMap {
+	return template.FuncMap{
+		"randInt":      randInt,
```

Comment:
I'd suggest making these function names capitals (to show clearly they're constants).

Reply:
Done.